kit/layout/src/UILayoutConstant.ts

```diff
@@ -4,6 +4,7 @@ const GRID_LINE_HEIGHT = 4;
 
 export const UILayoutConstant = {
     iconSize: 24,
+    tapZoneSize: 48,
```

Comment:
How much do we need this global parameter? It seems to me that the size of the tap zone should depend on the size of the component. It seems to me it is better to define this constant locally, in the folder of this component.
What do you think @savelichalex ?

Reply:
yeah, actually it might depends